infra/nodes/akka-node-001.json

```diff
@@ -0,0 +1,32 @@
+{
+    "chef_environment": "_default", 
+    "ipaddress-internal": "10.132.0.7", 
+    "ipaddress": "104.155.50.175", 
+    "name": "akka-node-001", 
+    
+    "akka": {
+      "seed_nodes": "[\"10.132.0.7\"]",
+      
+      "multi_node": [
+        "10.132.0.7",
+        "10.132.0.7",
+        "10.132.0.7",
+        "10.132.0.7",
+        "10.132.0.7",
+        "10.132.0.7",
+        "10.132.0.8"
+      ]
```

Comment:
this is where you would change the values of the file that multi node tests use. 
The file gets generated from this